### PR TITLE
Update command descriptions

### PR DESCRIPTION
**Pull request by carolinamenezes**
#### What is the purpose of this pull request?

_Update descriptions of Toolbelt's default commands._

#### Types of changes
- [ ] Refactor (non-breaking change that only makes the code better)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

#### Chores checklist
- [ ] Update `CHANGELOG.md`

### CODE REVIEW

File: src/commands/deprecate.ts [hunk 1]
```diff
@@ -3,21 +3,23 @@ import { flags as oclifFlags } from '@oclif/command'
...
+    yes: oclifFlags.boolean({ description: 'Answers yes to all prompts.', char: 'y', default: false }),
   }
 
   static strict = false
 
   static args = [
-    { name: 'appId', required: false },
-    { name: 'ithAppId', required: false, multiple: true },
+    { name: 'appId', required: false, description: `Name and version of the app ${ColorifyConstants.ID('({vendor}.{appname}@{x.x.x})')} to deprecate.` },
+    { name: 'ithAppId', required: false, multiple: true, description: `Names and versions of the multiple apps ${ColorifyConstants.ID('({vendor}.{appname}@{x.x.x})')} to deprecate.` },
```
Comment (dhasuda) [inline src/commands/deprecate.ts:22]:
```suggestion
    { name: 'ithAppId', required: false, multiple: true, description: `Names and versions of multiple apps ${ColorifyConstants.ID('({vendor}.{appname}@{x.x.x})')} to deprecate.` },
```

File: src/commands/link.ts [hunk 2]
```diff
@@ -3,30 +3,32 @@ import { flags as oclifFlags } from '@oclif/command'
 import { CustomCommand } from '../api/oclif/CustomCommand'
 import { appLink } from '../modules/apps/link'
 
+import { ColorifyConstants } from '../api/constants/Colors'
+
 export default class Link extends CustomCommand {
-  static description = 'Start a development session for this app'
+  static description = `Locally syncs the app in the current directory with the ${ColorifyConstants.ID('development workspace')} in use.`
```
Comment (dhasuda) [inline src/commands/link.ts:9]: I don't understand the `locally` in the description. The link happens in the cloud, everything workspace related happens in the cloud as far as I know

File: src/commands/link.ts [hunk 3]
```diff
@@ -3,30 +3,32 @@ import { flags as oclifFlags } from '@oclif/command'
...
+      description: `Starts a development session in the specified ${ColorifyConstants.ID('account')}. Must be paired with the ${ColorifyConstants.COMMAND_OR_VTEX_REF('--workspace')} flag.`,
       required: false,
       dependsOn: ['workspace'],
     }),
-    clean: oclifFlags.boolean({ char: 'c', description: 'Clean builder cache', default: false }),
+    clean: oclifFlags.boolean({ char: 'c', description: 'Cleans builder cache', default: false }),
     setup: oclifFlags.boolean({
       char: 's',
-      description: 'Setup typings before linking [see vtex setup --help]',
+      description: `Sets up typing definitions before linking the app [see ${ColorifyConstants.COMMAND_OR_VTEX_REF('vtex setup --help')}].`,
```
Comment (dhasuda) [inline src/commands/link.ts:24]: Here we can say that the setup happens locally

File: src/commands/link.ts (hunk 2 above)
Comment (dhasuda) [reply src/commands/link.ts:9]: Perhaps I just mismatched the command. I understood that it's related to the `vtex link` command

File: src/commands/release.ts [hunk 4]
```diff
@@ -1,16 +1,18 @@
 import { CustomCommand } from '../api/oclif/CustomCommand'
 import appsRelease, { releaseTypeAliases, supportedReleaseTypes, supportedTagNames } from '../modules/release'
 
+import { ColorifyConstants } from '../api/constants/Colors'
+
 export default class Release extends CustomCommand {
   static description =
-    'Bump app version, commit and push to remote. Only for git users. The first option can also be a specific valid semver version'
+    '(Only for git users.) Bumps the app version, commits, and pushes to remote the app in the current directory.'
```
Comment (dhasuda) [inline src/commands/release.ts:8]:
```suggestion
    '(Only for git users) Bumps the app version, commits, and pushes to remote the app in the current directory.'
```

File: src/commands/setup.ts [hunk 5]
```diff
@@ -4,29 +4,29 @@ import { CustomCommand } from '../api/oclif/CustomCommand'
...
 export default class Setup extends CustomCommand {
-  static description = 'Setup development enviroment'
+  static description = 'Sets up typings and tools for the current development environment.'
 
   static flags = {
     ...CustomCommand.globalFlags,
     'ignore-linked': oclifFlags.boolean({
       char: 'i',
-      description: 'Add only types from apps published',
+      description: 'Sets up types from published apps, and ignores types from linked apps.',
```
Comment (dhasuda) [inline src/commands/setup.ts:13]:
```suggestion
        description: 'Sets up types from published apps while ignoring types from linked apps.',
```

File: src/commands/unlink.ts [hunk 6]
```diff
@@ -3,21 +3,23 @@ import { flags as oclifFlags } from '@oclif/command'
...
+    all: oclifFlags.boolean({ char: 'a', description: 'Unlinks all apps.', default: false }),
   }
 
   static strict = false
 
   static args = [
-    { name: 'appId', required: false },
-    { name: 'ithAppId', required: false, multiple: true },
+    { name: 'appId', required: false, description: 'Name of the app to unlink.' },
+    { name: 'ithAppId', required: false, multiple: true, description: 'Names of the multiple apps to unlink.' },
```
Comment (dhasuda) [inline src/commands/unlink.ts:22]:
```suggestion
    { name: 'ithAppId', required: false, multiple: true, description: 'Names of multiple apps to unlink.' },
```

File: src/commands/link.ts (hunk 2 above)
Comment (carolinamenezes) [reply src/commands/link.ts:9]: I thought about local development. But yes, it's a bit confusing.  
What do you think about: 
Syncs the app in the current directory with the ${ColorifyConstants.ID('development workspace')} in use.
?

File: src/commands/link.ts (hunk 3 above)
Comment (carolinamenezes) [reply src/commands/link.ts:24]:
```suggestion
      description: `Locally sets up typing definitions before linking the app [see ${ColorifyConstants.COMMAND_OR_VTEX_REF('vtex setup --help')}].`,
```

File: src/commands/link.ts (hunk 2 above)
Comment (VerasThiago) [reply src/commands/link.ts:9]: Good Idea

**Review by VerasThiago — APPROVED** [review]
LGTM

**Pull request merged**